apps/Gruntfile.js

```diff
@@ -354,6 +354,7 @@ module.exports = function (grunt) {
     'levelbuilder_studio': './src/code-studio/levelbuilder_studio.js',
     'levelbuilder_gamelab': './src/code-studio/levelbuilder_gamelab.js',
     'levelbuilder_applab': './src/code-studio/levelbuilder_applab.js',
+    'levelbuilder_edit_script': './src/code-studio/levelbuilder_edit_script.js',
```

Comment:
Yeah, I believe this should end up in our core file (called essential.js rather than minimal.js since "min" was being stripped)